jspdf.js

```diff
@@ -380,10 +444,29 @@ var jsPDF = (function(global) {
           }
         }
         out('>>');
+        if (pdfVersion == '1.4') {
```

Comment:
Why dont we PubSub this?

Reply:
Please explain what you mean by PubSub.  Thanks.